src/Components/Data.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {
+  Flex,
+  Image,
+  Stack,
+  Text,
+  Heading,
+  Divider,
+  Box,
+  Tooltip,
+  Spacer,
+  useBreakpointValue,
+} from '@chakra-ui/react';
+import { Link } from 'react-router-dom';
+import { StarIcon } from '@chakra-ui/icons';
+
+const Data = ({ data, loading }) => {
+  const skeletonArray = Array.from({ length: 80 });
+  const isMobile = useBreakpointValue({ base: true, md: false });
+
+  return (
+    <Box>
+      <Flex
+        direction={isMobile ? 'column' : 'row'}
+        justifyContent={isMobile ? 'center' : 'flex-start'}
+        alignItems={isMobile ? 'center' : 'flex-start'}
+        flexWrap="wrap"
+        
+      >
+        {loading
+          ? skeletonArray.map((_, index) => (
+              <Box
+                key={index}
+                maxW="sm"
+                flexBasis="19%"
+                m={4}
+                p={4}
+                borderWidth="1px"
+                bg="white"
+                borderRadius="lg"
+                overflow="hidden"
+                boxShadow="md"
+              >
+              </Box>
+            ))
+          : data.map((item, index) => (
+              <Box
+                key={index}
+                maxW="xs"
+                flexBasis={isMobile ? '100%' : '30%'}
+                m={4}
+                 bg="ghostwhite"
+                p="20px"
+                transition="transform 0.2s"
+                _hover={{
+                  transform: 'scale(1.05)',
+                  cursor: 'pointer',
+                  bg: 'white',
+                  boxShadow: 'md',
+                 
+                }}
+              >
+                <Tooltip label={item.productTitle}>
+                  <Link to={`/product/${item._id}`}>
+                    <Flex justify="center" align="center"  bg="ghostwhite" h="100px">
+                      <Image
+                        src={item.HeadImage}
```

Comment:
```suggestion
                        src={item.headImage}
```
please ask backend to change the case

Reply:
done